doc/api/tracing.md

```diff
@@ -23,8 +23,12 @@ The available categories are:
 * `node.environment` - Enables capture of Node.js Environment milestones.
 * `node.fs.sync` - Enables capture of trace data for file system sync methods.
 * `node.perf` - Enables capture of [Performance API] measurements.
+  * `node.perf.event_loop` - Enables capture of event loop delay information
+    when the `perf_hooks.monitorEventLoopDelay()` API is used.
   * `node.perf.usertiming` - Enables capture of only Performance API User Timing
     measures and marks.
+  * `node.perf.reqwrap` - Enables capture of ReqWrap Latency information when
```

Comment:
Since the outer list is sorted, maybe it is worth to sort this `node.perf...` sublist as well?